app/components/events/index.hbs

```diff
@@ -0,0 +1,26 @@
+<Header />
```

Comment:
Assuming we want a header to show up on ALL our routes, I suggest putting this in the application.hbs, so that we can leverage only writing it once instead of explicitly adding it in every route at the top

Reply:
Done!. Please check